site/content/coredns/_index.md

```diff
@@ -0,0 +1,140 @@
+---
+title: coredns
+---
+
+## Overview
+
+
```

Comment:
CoreDNS mixin which provides Grafana dashboard and Prometheus Alerts, the mixin was introduced in (Kubernetes Node Local DNS Cache blogpost)[https://povilasv.me/kubernetes-node-local-dns-cache/] to better help users monitor CoreDNS in Kubernetes. Mixin can also be used to monitor standalone CoreDNS instance without any orchestrators.

Reply:
Added :)
Sorry for delay

Reply:
> Mixin can also be used to monitor standalone CoreDNS instance without any orchestrators.

Yes, but by default it references k8s. More in https://github.com/povilasv/coredns-mixin/pull/5

Reply:
That PR got merged ;)